perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2454,6 +2454,13 @@ perun_policies:
     include_policies:
       - default_policy
 
+  createSponsoredMembersFromCSV_Vo_String_List<String>_User_policy:
+    policy_roles:
+      - SPONSOR: Vo
+        SELF: User
```

Comment:
Shouldn't here be also REGISTRAR role since it is mentioned in rpc javadoc?

Reply:
It doesn't really make sense, I will edit the javadoc.